### PR TITLE
FMC methods added to add Waypoint and clear all entries provided

### PR DESCRIPTION
**Pull request by kamgolabek**

### CODE REVIEW

File: extplane-plugin/xplaneplugin.cpp [hunk 1]
```diff
@@ -329,9 +329,36 @@ bool XPlanePlugin::loadSituation(QString sitFileLocation) {
...
 }
 
+/**
+ * @brief XPlanePlugin::addFMSEntryLatLon
+ * @param fmsEntryLine - string in format "id,latitude,longitude,altitude"
+ * example: "1,50.0267,8.51,198"
+ */
+void XPlanePlugin::addFMSEntryLatLon(QString fmsEntryLine){
+     QStringList params = fmsEntryLine.split(",", QString::SkipEmptyParts);
+     int id = params.value(0).toInt();
```
Comment (vranki) [inline extplane-plugin/xplaneplugin.cpp]: You could check for invalid values here before continuing.. But i guess it's ok, it'll just put zeros in place of invalid values given and this should not crash.

Comment (vranki) [reply extplane-plugin/xplaneplugin.cpp]: Docs also say that "There is a max of 100 waypoints in the flight plan. " so please add a check to limit max number of entries to 100. Also looks like entry -1 would go through and could cause a crash. So check that id values are between 0 and 100.

File: README.md [hunk 2]
```diff
@@ -263,9 +263,13 @@ Command identifiers are strings that look like datarefs.
...
 
 ### Situations ###
-* ** sit "{situationFileLocation}" ** situation file location relative to XPlane root folder <br />
+* **sit "{situationFileLocation}"** situation file location relative to XPlane root folder <br />
  (e.g.,  sit "Output/situations/SampleSit.sit" )
 
+### FMC ###
+* **fms_wpt_entry id,latitude,longitude,altitude** Inserts waypoint to FMC. (e.g., "fms_wpt_entry 1,50.0267,8.51,32000") <br />
+* **clearAllFmsEntries** Clears all entries in FMC. 
+
```
Comment (vranki) [inline README.md]: The actual command seems to be fms_clear_entries

**Comment by kamgolabek** [comment]
There is no sens to check id in this case. FMC behaves liek that :  if id is less than zero, it will just ignore it, and every id bigger than actual waypoints count will be putted at the end

**Comment by vranki** [comment]
Merged, thanks for contributing!

**Pull request merged**